introduction.md

```diff
@@ -1,2 +1,3 @@
-My owsom introduction
-javascript
\ No newline at end of file
+*Git
+*JavaScript
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?